src/main/kotlin/net/atos/zac/util/BSNValidator.kt

```diff
@@ -0,0 +1,26 @@
+/*
+ * SPDX-FileCopyrightText: 2025 Lifely
+ * SPDX-License-Identifier: EUPL-1.2+
+ */
+package net.atos.zac.util
+
+object BSNValidator {
```

Comment:
Why an object? Could just be plain functions I think?

And at that maybe even extension functions on String? `fun String.validateBsn(): {..}`

Reply:
Changed to extension fn